src/components/Message/Message.tsx

```diff
@@ -1048,9 +1057,11 @@ const MessageWithContext = <
                 targetedStyle,
               ]}
             />
-            <MessageProvider value={messageContext}>
-              <MessageSimple />
-            </MessageProvider>
+            <MessagesProvider value={messagesContext}>
```

Comment:
Lets add some nice comment here for future purpose, why we have `MessagesProvider` here!!

Reply:
Can do.